src/pymmcore_plus/mda/handlers/_tiff_sequence_writer.py

```diff
@@ -132,16 +133,42 @@ def sequenceStarted(self, seq: useq.MDASequence) -> None:
             # make directory and write metadata
             self._seq_meta_file.write_text(seq.json(exclude_unset=True, indent=4))
 
+    def _get_full_sequence_axis(self, sequence: useq.MDASequence) -> tuple[str, ...]:
```

Comment:
pull this off into a convenience method (since it's not related to this write, but could be more generally useful)

Reply:
(and rename to `get_full sequence_axes`)